orbit/db2.py

```diff
@@ -38,7 +38,7 @@ def get_all() -> list[Self]:
 
     def insert_new(token: str, username: str, expiry: str):
         Session.create(token=token, username=username, expiry=expiry)
-    
```

Comment:
its good that you kept this a separate commit (since it makes what I am about to suggest easier), but it should probably just be squashed into the commit that actually introduced the whitespace error to tidy up the log. When you need to fix something that is already merged, ok then it needs to be its own commit, but within your mr, you can use rebase to make it as if you never made these kind of mistakes